tests/foreman/cli/test_reporttemplates.py

```diff
@@ -110,15 +100,21 @@ def local_ak(local_org, local_environment, local_content_view):
         {
             'lifecycle-environment-id': local_environment['id'],
             'content-view': local_content_view['name'],
-            'organization-id': local_org['id'],
+            'organization-id': module_entitlement_manifest_org.id,
             'auto-attach': False,
         }
     )
 
 
 @pytest.fixture(scope='module')
-def local_subscription(local_org, local_ak):
-    subscription = Subscription.list({'organization-id': local_org['id']}, per_page=False)[0]
+def local_subscription(module_entitlement_manifest_org, local_ak):
+    subscription = next(
+        sub
+        for sub in Subscription.list(
+            {'organization-id': module_entitlement_manifest_org.id}, per_page=False
+        )
+        if sub['name'] == DEFAULT_SUBSCRIPTION_NAME
+    )
```

Comment:
Why this version over something like this?
```python
for subscription in Subscription.list(
    {'organization-id': module_entitlement_manifest_org.id}, per_page=False
):
    if subcription['name'] == DEFAULT_SUBSCRIPTION_NAME:
        break
```

Reply:
I frankly don't remember why I landed on this particular construction. I think your version is both more concise and more readable, so I'm happy change this. I'm running the report templates locally with your code, with two of the three passing so far. Assuming the third one passes as well, I'll make the switch on Monday.

Reply:
...and the third one passed as well just as I posted that comment. Just pushed the change. Thanks for the suggestion!